src/components/stats/VolumeChart.tsx

```diff
@@ -211,12 +218,14 @@ const useVolumeMetrics = (): ({ timestamp: number } & Record<
             // TODO too repetitive; shouldn't be doing this anyway;
             // the daily values should be on the subgraph
             minted -= parseFloat(prev.cumulativeMinted.simple);
-            deposited -= parseFloat(
-              prev.savingsContracts[0].cumulativeDeposited.simple,
-            );
-            withdrawn -= parseFloat(
-              prev.savingsContracts[0].cumulativeWithdrawn.simple,
-            );
+            // I commented these out, do we need them? :thinking:
```

Comment:
We do need them; it subtracts the previous value to get the difference between them, like a slice that represents the current day.